checkov/terraform/checks/resource/aws/AutoScalingTagging.py

```diff
@@ -14,9 +15,12 @@ def scan_resource_conf(self, conf):
         """
             Looks for tag or tags
         """
-
         if "tag" in conf.keys() or "tags" in conf.keys():
             return CheckResult.PASSED
-        return  CheckResult.FAILED
+        return CheckResult.FAILED
+
+    def get_evaluated_keys(self) -> List[str]:
+        return ['tag', 'tags']
```

Comment:
@YaaraVerner Isn't it either-or? Either `tag` was used, or `tags`, right?

Reply:
it is. but the key doesn't exist it ignored